src/gatsby/src/styles/listPage.scss

```diff
@@ -181,10 +180,5 @@
       width: 24px;
       height: 24px;
     }
-
-    @media screen and (max-width: $mdBreakpoint) {
-      margin-top: -8px;
-      margin-bottom: 12px;
-    }
   }
```

Comment:
We should also revert the CSS changes that were made in this PR: 
https://github.com/bcgov/bcparks.ca/pull/1143